examples/end-to-end-examples/support_chatbot/mcli_yamls/finetune/finetune_30b_chat.yaml

```diff
@@ -0,0 +1,219 @@
+name: mpt-30b-PyPi_composer_chatv2
+
+scheduling:
+  resumable: true
+  priority: low
+
+compute:
+  gpus: 64  # Number of GPUs to use
+
+  ## These configurations are optional
+  # cluster: r0z0 # Name of the cluster to use for this run
+  gpu_type: h100_80gb # Type of GPU to use.
+
+integrations:
+# Clone and install the llm-foundry repo so we can run scripts from it
+- integration_type: git_repo
+  git_repo: mosaicml/llm-foundry
+  git_branch: davis/lion8b-v2
```

Comment:
probably don't want to include the lion8b branch as the final thing to run with.